kura/org.eclipse.kura.net.admin/src/main/java/org/eclipse/kura/net/admin/visitor/linux/IfcfgConfigReader.java

```diff
@@ -287,11 +266,20 @@ private File getIfcfgFile(String interfaceName) {
         return new File(fileName);
     }
 
+    private Properties getKuraPropertiesFromIfaceConfigFile(String ifaceName, File ifcfgFile) throws KuraException {
+        Properties props;
+        // found our match so load the properties
+        if (isDebian()) {
+            props = parseDebianConfigFile(ifcfgFile, ifaceName);
+        } else {
+            props = parseRedhatConfigFile(ifcfgFile, ifaceName);
+        }
+        return props;
+    }
+
     private Properties parseRedhatConfigFile(File ifcfgFile, String interfaceName) {
```

Comment:
Is it make sense to move also this to the IfcfgConfig class? I know that it is not used in writer, but maybe "logically" can be moved there. What do you think?

Reply:
done